relay-server/src/endpoints/common.rs

```diff
@@ -295,6 +294,7 @@ pub fn cors(app: ServiceApp) -> CorsBuilder<ServiceState> {
             "authentication",
             "authorization",
             "content-encoding",
+            "transfer-encoding",
```

Comment:
This change is somewhat unrelated. I noticed that the `transfer-encoding` header was missing from CORS, but it is actually allowed and even required for Crashpad uploads. For consistency, I opted to add it here now.